javascript/challenges/queueWithStacks/queue-with-stacks.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+class PseudoQueue {
+
+  constructor(value) {
+    this.value = value;
+  }
+
+  enqueue() {
+    let firstStack = new Stack();
+    let secondStack = new Stack();
+    let node = new Node ();
+    let enqueueNode = new Node ();
+    node = firstStack.top;
+    while (firstStack.top.value !== null) {
+      secondStack.push(this.firstStack.pop);
+    }
+    enqueueNode.push(firstStack);
+    while (secondStack.top.value !== null) {
+      firstStack.push(secondStack.pop);
+    }
+    return firstStack;
+  };
+
+  dequeue() {
+    let firstStack = new Stack();
+    let secondStack = new Stack();
```

Comment:
Something that you could think about doing here is pushing everything from stack to another, that's going to cause everything to reverse the order, and then from there you will pop off